keel-core/src/main/kotlin/com/netflix/spinnaker/keel/actuation/ResourceActuator.kt

```diff
@@ -171,7 +171,7 @@ class ResourceActuator(
           }
         }
       } catch (e: ResourceCurrentlyUnresolvable) {
-        log.warn("Resource check for {} failed (hopefully temporarily) due to {}", id, e.message)
+        log.warn("Resource check for {} failed (hopefully temporarily) due to {}", id, e.message, e)
```

Comment:
Ensure stack trace is logged in this case, so that exception is captured by our exception monitoring system.